kornia/losses/ms_ssim_l1.py

```diff
@@ -96,16 +97,21 @@ def _fspecial_gauss_2d(self, size, sigma):
         gaussian_vec = self._fspecial_gauss_1d(size, sigma)
         return torch.outer(gaussian_vec, gaussian_vec)
 
-    def forward(self, yhat: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
+    def forward(self, yhat: torch.Tensor, y: torch.Tensor, gpu_idx: int = 0) -> torch.Tensor:
```

Comment:
We don’t usually do like this. Instead, before line 116 I suggest,
`g_masks = self.g_masks.to(yhat)`

Reply:
done